### ISSUE TITLE
railway doesnt work

### ISSUE THREAD

**Issue by noureldinz3r0** [n4ze3m/dialoqbase#30]
Hi, it keeps crashing on railway
![image](https://github.com/n4ze3m/dialoqbase/assets/79485227/d071462e-938f-4e2d-bfe9-d3942f7372e2)
![image](https://github.com/n4ze3m/dialoqbase/assets/79485227/7777b267-8da3-4839-97a7-4f4a560b2253)

**Comment by n4ze3m** [comment]
I'm sorry, it's a database connection URL issue. Did you paste the Supabase URL correctly? You can watch a quick demo [here](https://twitter.com/n4ze3m/status/1668208861663354882)

**Comment by noureldinz3r0** [comment]
yes i did, let me try again maybe i did it wrong
do i need to setup redis? cuz i donno how :D

**Comment by n4ze3m** [comment]
No, you don't need to set up Redis since I have already configured it with Railway's template

**Comment by noureldinz3r0** [comment]
im trying to redeploy after i got the supabase uri again.

**Comment by n4ze3m** [comment]
> im trying to redeploy after i got the supabase uri again.

Make sure you replace `[YOUR-PASSWORD]` with the correct database password :)

**Comment by noureldinz3r0** [comment]
should the password be between [] ? cuz maybe that's y, i did it without the [] :D ?

**Comment by noureldinz3r0** [comment]
ill re do the whole thing maybe i memorized the pass wrong

**Comment by n4ze3m** [comment]
Suppose your password is 'mysupercretpassword,' and you copied the

Supabase connection URL `postgresql://postgres:[YOUR-PASSWORD]@db.somehost.supabase.co:5432/postgres`
 
You need to replace it with `postgresql://postgres:mysupercretpassword@db.somehost.supabase.co/postgres`

**Comment by noureldinz3r0** [comment]
It worked, the issue was that the password had a hashtag, and it caused a problem for a reason
